airgun/views/computeresource.py

```diff
@@ -320,7 +320,9 @@ class ComputeResourceVMwareProfileStorageItem(GenericRemovableWidgetItem):
     controller = FilteredDropdown(
         locator=".//div[@class='controller-header']//div[contains(@class, 'form-control')]"
     )
-    remove_button = Text(".//button[contains(@class, 'remove-controller')]")
+    remove_button = Text(
+        ".//button[contains(concat(' ', @class, ' '), ' btn-remove-controller ')]"
```

Comment:
These concats shouldn't be necessary. Did the class just change from `remove-controller` to `btn-remove-controller`?

Reply:
Ah maybe you do. annoying!